src/test/java/com/epam/tc/hw1/SubTest.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+
+public class SubTest extends AbstractTest {
+
+    @Test
+    public void subTest() {
+        double res = calculator.sub(7, 2);
+        assertThat(res).isEqualTo(5);
+    }
```

Comment:
Use dataproviders

_6.	All test data should be provided to test through the DataProviders_

Reply:
I removed simply tests and expanded the data set in the existing ones through the DataProviders